web/package.json

```diff
@@ -53,12 +54,16 @@
     "@types/react-dom": "18.0.5",
     "@types/react-table": "^7.7.11",
     "@vitejs/plugin-react": "^1.3.2",
+    "aws-amplify": "^4.3.27",
```

Comment:
can you remove this import? We have `@aws-amplify/auth` which you can use instead!